app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -173,6 +173,7 @@ function replaceActionButts(el) {
 }
 
 function warmNewCommentsArea() {
+  if (window.location.pathname === "/notifications") { return }
```

Comment:
This guard clause is needed because when users create a reply within `/notifications`, this function will be trying to fetch `/stories/warm_comments/notifications` which is invalid and will fail specs.

Reply:
We might want to leave your above note as a comment :) I will definitely forget why we're doing this in the future 😛

Reply:
```suggestion
  // Ignore when comment is created on /notifications.
  if (window.location.pathname === "/notifications") { return }
```